pgml-extension/pgml_rust/src/orm/snapshot.rs

```diff
@@ -279,7 +279,7 @@ impl Snapshot {
                 (num_rows as f32 * self.test_size).round() as usize
             };
             let num_train_rows = num_rows - num_test_rows;
-            if num_train_rows <= 0 {
+            if num_train_rows == 0 {
```

Comment:
Clippy never heard of off-by-one errors? == 0 always made me paranoid.

Reply:
it's unsigned. Can't be < 0.